CHANGELOG.md

```diff
@@ -43,6 +45,13 @@ This change was introduced in [pull request #4566: Use Character count `formGrou
 
 ### Fixes
 
+We've fixed an upstream issue in the cssnano package that caused elements with transparency to render incorrectly in Internet Explorer 11. This affected the pre-compiled CSS files we provide on [npm](https://www.npmjs.com/package/govuk-frontend) and in [GitHub releases](https://github.com/alphagov/govuk-frontend/releases). These were fixed in:
```

Comment:
Just a suggestion, but do we say "XXX npm package" in links?

Worth saying which releases were affected? From [v5.0.0-internal.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.0.0-internal.0) to [v5.2.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.2.0)

```suggestion
We've fixed an upstream issue in the [cssnano npm package](https://www.npmjs.com/package/cssnano) that caused elements with transparency to render incorrectly in Internet Explorer 11. This affected the pre-compiled CSS files in the [GOV.UK Frontend npm package](https://www.npmjs.com/package/govuk-frontend) and [GitHub releases](https://github.com/alphagov/govuk-frontend/releases). These were fixed in:
```